api/odilo.py

```diff
@@ -116,6 +146,9 @@ def get_patron_credential(self, patron, pin):
         def refresh(credential):
             return self.get_patron_access_token(credential, patron, pin)
 
+        return self._credential_lookup(patron, refresh)
```

Comment:
Should this be `self._patron_credential_lookup`?

Reply:
The fact that this error didn't cause big problems rang an alarm bell for me. I investigated and it turns out that the Odilo API actually has no concept of a "patron token". The API doesn't issue patron tokens and when it acts on behalf of a patron it uses the global token. The 'patron token' stuff is just something that was copied from the Overdrive code this was based on.

I filed https://jira.nypl.org/browse/SIMPLY-676 to clean up the code, but the upshot for purposes of this branch is that the Odilo self-test doesn't need to test functionality that doesn't exist.